.github/workflows/gpu_test.yml

```diff
@@ -13,8 +13,7 @@ jobs:
       run: |
         conda create --name alphapept_gpu python=3.8 -y
         conda activate alphapept_gpu
-        pip install .
-        pip install .
+        pip install ".[gui-stable]"
```

Comment:
If you only want to test with stable requirements, you should use `pip install ".[stable,gui-stable]"`

Reply:
Check this for all other files as well